clusters/base/kepler.yaml

```diff
@@ -20,15 +20,21 @@ metadata:
 spec:
   interval: 24h
   # Values.yaml updated. Refer: https://fluxcd.io/flux/components/helm/helmreleases/#inline-values
+
   values:
     serviceMonitor:
       enabled: "true"
       namespace: "monitoring"
+    nodeSelector:
+      kubernetes.io/os: linux
+      cncf-project: wg-green-reviews
+      cncf-project-sub: internal
```

Comment:
This isn't needed. We should run kepler on all nodes.

Reply:
okay